app/views/users/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<section class="user">
+  <div class="in_out">
+    <p class="welcome_page">Welcome, <%= @users.name %></p>
+
+    <p><%= link_to "Log out", destroy_user_session_path, data: { "turbo-method":
+    :delete } %></p>
+  </div>
+ 
+  <div class="logo">
+    <p class="app_name">E-neza Transaction</p>
+  </div>
+
+  <button class="flex-end-add">
+    <%= link_to "Start", categories_path, class: "add-cate" %>
```

Comment:
- [ ] Although you are not to use this design, I couldn't see the text that says "add category" in your button as you did in your demo.